### DISCUSSION TITLE
Variable positional arguments

### DISCUSSION THREAD

**Discussion by sylvorg** [coady/multimethod#116] [Q&A] [answered]
Hello! Sorry, it's me again! 😅

Is there any way to implement a variable number of positional arguments, like in the following example:
```python
from multimethod import multimethod

@multimethod
def call():
    print("Hello, world!")

@multimethod
def call(name):
    print(f"Hello, {name}!")

call() # Hello, world!
call("Foo") # Hello, Foo!
```

Thank you kindly for the help, and sorry again for the multiple issues!

**Comment by coady** [comment, marked as answer]
`multimethod` supports variable arguments, but there are no types in the example. The pythonic way to do this kind of "overloading" is to use defaults.
```python
def call(name="world"):
    print(f"Hello, {name}!")
```

**Comment by sylvorg** [comment]
Actually, speaking of which, how do you use `multidispatch` with defaults / keyword arguments? And are they the same thing?
